scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/RevioPacBioProcessor.java

```diff
@@ -410,4 +368,24 @@ private boolean isRunComplete(File runDirectory, int smrtCellCount) {
       throw new RuntimeException(e);
     }
   }
+
+  /**
+   * Check for presence of Transfer_Test file and grab start time
+   *
+   * @param runDirectory which we are currently processing
+   * @param dto PacBioNotificationDto which we will modify
+   */
+  private void getStartTimeFromTransferTests(File runDirectory, PacBioNotificationDto dto) {
+    try (Stream<Path> stream = Files.walk(runDirectory.toPath())) {
+      // Need to take the list of file paths and get the earliest creation time
+      Optional<Path> earliestCreatedFilePath =
+          stream
+              .filter(Files::isRegularFile)
+              .filter(file -> TRANSFER_TEST.test(String.valueOf(file.getFileName())))
+              .min(Comparator.comparing(filePath -> getFileCreationTime(filePath.toFile())));
+      dto.setStartDate(getFileCreationTime(earliestCreatedFilePath.orElse(null).toFile()));
```

Comment:
It looks like you're calling `getFileCreationTime` twice here; can you think of a way to simplify this?